### ISSUE TITLE
Automated testing of the pymt Jupyter Notebooks

### ISSUE THREAD

**Issue by gantian127** [csdms/pymt#126]
There is a need for the automated testing of the pymt Jupyter Notebooks. This will help make sure the version update won't break the notebooks.

**Comment by mdpiper** [comment]
A candidate solution: https://github.com/nteract/testbook. From its README:

Consider the following code cell in a Jupyter Notebook:

```python
def func(a, b):
   return a + b
```

You would write a unit test using `testbook` in a Python file as follows:

```python
import testbook


@testbook.testbook('/path/to/notebook.ipynb', execute=True)
def test_func(tb):
   func = tb.ref("func")

   assert func(1, 2) == 3
```

**Comment by mdpiper** [comment]
Since we currently use pytest for testing, [pytest-notebook](https://pytest-notebook.readthedocs.io/en/latest/) may be a better candidate.

**Comment by mdpiper** [comment]
Fixed with #133.
